latch_cli/nextflow/workflow.py

```diff
@@ -187,18 +199,23 @@ def get_flag(name: str, val: Any) -> List[str]:
         return [flag, str(val)]
 
 
-def generate_nextflow_config(pkg_root: Path):
+def generate_nextflow_config(
+    pkg_root: Path, process_executor: Optional[NextflowProcessExecutor]
+):
+    if process_executor is None:
+        process_executor = NextflowProcessExecutor.k8s
+
     config_path = Path(pkg_root) / "latch.config"
-    config_path.write_text(dedent("""\
-        process {
-            executor = 'k8s'
-        }
+    config_path.write_text(dedent(f"""\
```

Comment:
why make a config file instead of passing these in as command line args?

Reply:
aws.anonymous couldn't be pass in as a command line argument (or at least i couldn't figure out how to

Reply:
mmk